doc/content/gateways/lora-basics-station/cups.md

```diff
@@ -29,6 +29,7 @@ If you have not already created one, follow instructions for [Creating a Gateway
 In your gateway configuration, set the following fields:
 
 CUPS URI: `https://<server-address>:443`
-CUPS Trust: Amazon Root CA 1
+
 CUPS Key: `<gateway-api-key>`
-You can download the CUPS Trust from [here](https://ssl-tools.net/subjects/4f59a39453cfb9559e7f6bd8c54da53da642b714).
+
+CUPS Trust: Use the CA certificate of your trust provider
```

Comment:
Can you create an issue for CH certificates and assign it to me?

Reply:
#2712